docs/shared/guides/react-native.md

```diff
@@ -225,7 +225,7 @@ Run:
 To generate a new component inside `shared-ui-layout` run:
 
 ```shell
-npx nx g @nx/react-native:component layout --directory=libs/shared-ui-layout/src/lib/layout --export
+npx nx g @nx/react-native:component libs/shared-ui-layout/src/lib/layout/layout --export
```

Comment:
Yeah, it's keeping the behavior the same as before `layout/layout.tsx`, otherwise it won't be nested. Obviously, it's a example so we can show whatever we want.